.docker/ci/Dockerfile

```diff
@@ -8,31 +8,42 @@ ENV TERM xterm
 
 # Setup catkin workspace
 ENV CATKIN_WS=/root/ws_moveit
-RUN mkdir -p $CATKIN_WS/src
```

Comment:
Having an extra RUN command here, creates an additional layer, which you wanted to avoid.
As pointed out by gavanderhoorn in https://github.com/ros-planning/moveit/pull/902#discussion_r189317983, WORKDIR will create the folder if not existing yet.

Reply:
sure